sammo.py

```diff
@@ -3,18 +3,48 @@
 __contact__ = "info@hytech-imaging.fr"
 __copyright__ = "Copyright (c) 2021 Hytech Imaging"
 
-from .src.gui.session import SammoActionSession
+from .src.gui.session import SammoActionSession, ParentOfSammoActionSession
+from .src.gui.onOffEffort import SammoActionOnOffEffort, ParentOfSammoActionOnOffEffort
+from .src.core.session import SammoSession
 
 
-class Sammo:
+class Sammo(ParentOfSammoActionSession, ParentOfSammoActionOnOffEffort):
```

Comment:
We can't rely on this mechanism because we'll have a lot of buttons and I don't want to inherit from 10 classes :).